public/content/docs/mels-hack/index.en.md

```diff
@@ -0,0 +1,173 @@
+
+This layout is missing a part described earlier in the story:
+
+        The new computer had a one-plus-one
+        addressing scheme
+        in which each machine instruction,
+        in addition to the operation code
+        and the address of the needed operand,
+        had a second address that indicated where, on the revolving drum,
+        the next instruction was located.
+
+
+Thus, the bit layout of the instruction needs an additional component for the next address `(N)`. Its location doesn't affect the hack, as described in the story, but let's place it on the least significant bits, to be on the safe side: 
+
+            AAAXCCCNNN
+        MSB <----------> LSB
+   
+    
+We know that the `(A)` bits are lower than the `(C)` bits, because Nather later describes the overflow that Mel hijacked:
+
+        Instead, he would pull the instruction into a machine register,
+        add one to its address,
+        and store it back
+        ...
+        He had located the data he was working on
+        near the top of memory —
+        the largest locations the instructions could address —
+        so, after the last datum was handled,
+        incrementing the instruction address
+        would make it overflow.
+        The carry would add one to the
+        operation code, changing it to the next one in the instruction set:
+
+
+If incrementing the address span overflowed into the opcode span, then the bit order between them is established:
+
+            CCCXAAANNN
+        MSB <----------> LSB
+
+If the index register bit `(X)` is indeed between the two and turned on, then overflowing the `(A)` span will carry through `(X)` into the `(C)` span, incrementing it by one. The result:
+
+        a jump instruction.
+        Sure enough, the next program instruction was
+        in address location zero,
+        and the program went happily on its way.
+
+### The Unpleasant Truth
+
+All this is possible in theory, but we still don't know where the `JUMP` instruction takes its operand from. We know that the operand's value must be `0` and it should be ready. The address span `(A)` indeed contains `0`, but nowhere in the description of RPC-4000's architecture, does Nather mention an option of the data address field doubling as an instruction address. The `JUMP` instruction could take its value from some register, but that `0` would have to be stored there beforehand, providing a screaming clue that some operation was being set up.
+
+Not only did I gloss over this crucial step for 30 years - I also ignored the absence from this part of the story, of the unique addressing mode described above. The one that included the next instruction address in every instruction(!). Was that part of affected in the overflow? Was there any relationship between the `JUMP` opcode and the `(A)` or `(N)` fields? And what about that `(X)` bit? Its alleged location, between two bitfields, just seemed... wrong.
```

Comment:
"30 years" - tells a very long story in a way to short phrasing. When did you encounter the story? Present yourself through that question. It doesn't matter who you are, it matters who you are in relation to the story. Then these 30 years mean something more than just time.